### DISCUSSION TITLE
Unable to pull Elasticsearch image while running colima

### DISCUSSION THREAD

**Discussion by prashanttest-elastic** [abiosoft/colima#628] [Q&A]
I am able to pull Elasticsearch image without start colima. But after running `colima start --arch x86_64 --memory 4` command it shows following error `Error response from daemon: Get "https://docker.elastic.co/v2/": x509: certificate signed by unknown authority`. I am using mac os with m1 chip.

**Comment by abiosoft** [comment]
Are you still experiencing this?

**Reply by prashanttest-elastic** [reply]
> Are you still experiencing this?

yes

**Reply by prashanttest-elastic** [reply]
@abiosoft, do you have any idea ?

**Reply by abiosoft** [reply]
Are you behind a proxy?

**Reply by prashanttest-elastic** [reply]
No, I am not using any kind of proxy.

**Reply by prashanttest-elastic** [reply]
hey @abiosoft, I resolved the above issue by adding my registry name to the insecure-registries list. But now I am facing a new issue. now docker is able to pull the image but after pulling the image it got stuck and no any log will come.

**Reply by abiosoft** [reply]
Are you on an m1 device? I suspect the emulation is simply very slow. Is there no native arm64 docker image for elasticsearch?

**Reply by rfay** [reply]
The official elasticsearch image is multiplatform (amd64/arm64). https://hub.docker.com/_/elasticsearch/tags

DDEV uses it quite successfully with colima. https://github.com/ddev/ddev-elasticsearch
